homeassistant/components/cover/rflink.py

```diff
@@ -0,0 +1,93 @@
+"""
+Support for Rflink switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.rflink/
+"""
+import asyncio
+import logging
+
+
+from homeassistant.components.rflink import (
+    DATA_ENTITY_GROUP_LOOKUP, DATA_ENTITY_LOOKUP,
+    DEVICE_DEFAULTS_SCHEMA, DOMAIN, EVENT_KEY_COMMAND, CoverableRflinkDevice,
+    cv, vol)
+from homeassistant.components.cover import (
+    CoverDevice)
+
+from homeassistant.const import CONF_NAME, CONF_PLATFORM
+DEPENDENCIES = ['rflink']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+CONF_ALIASES = 'aliases'
+CONF_ALIASSES = 'aliases'
+CONF_GROUP_ALIASES = 'group_aliases'
+CONF_GROUP_ALIASSES = 'group_aliases'
+CONF_GROUP = 'group'
+CONF_NOGROUP_ALIASES = 'nogroup_aliases'
+CONF_NOGROUP_ALIASSES = 'nogroup_aliases'
+CONF_DEVICE_DEFAULTS = 'device_defaults'
+CONF_DEVICES = 'devices'
+CONF_AUTOMATIC_ADD = 'automatic_add'
+CONF_FIRE_EVENT = 'fire_event'
+CONF_IGNORE_DEVICES = 'ignore_devices'
+CONF_RECONNECT_INTERVAL = 'reconnect_interval'
+CONF_SIGNAL_REPETITIONS = 'signal_repetitions'
+CONF_WAIT_FOR_ACK = 'wait_for_ack'
+
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
I believe it's still used in some configuration and not yet fased out.
that's why 
from homeassistant.helpers.deprecation import get_deprecated 
is imported.